templates/cdk.master.leader.file-watcher.service

```
@@ -4,7 +4,8 @@ After=network.target
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/juju-run {{ unit }} /usr/local/sbin/cdk.master.leader.file-watcher.sh
+ExecStart=-/usr/bin/juju-exec {{ unit }} /usr/local/sbin/cdk.master.leader.file-watcher.sh
+ExecStart=-/usr/bin/juju-run {{ unit }} /usr/local/sbin/cdk.master.leader.file-watcher.sh
```

TIL of this `=-` syntax. neat!

---

my only concern here is that `ExecStart=-/usr/bin/bad_command` will just log any errors but the service won't fail.  So, there will be no reporting of this service ACTUALLY failing if juju decides to rename a command again or alias the command.  

if they alias it (ie -- both binaries are present on the system) -- well then, BOTH commands would be run -- effectively calling this process twice. 

i also considered

```suggestion
ExecStart= \
  /usr/bin/juju-exec {{ unit }} /usr/local/sbin/cdk.master.leader.file-watcher.sh || 
  /usr/bin/juju-run {{ unit }} /usr/local/sbin/cdk.master.leader.file-watcher.sh
```

---

hmm... something about the failed integration tests and these changes troubles me..